### PR TITLE
Python agent release v8.5.0 (Publish on Dec 8, 2022 with morning merge)

### PR DESCRIPTION
**Pull request by hmstepanek**
Python agent release v8.5.0

Changes:
* Add record_log_event to Python Agent's api.
* Add release notes & update EOL table to Python agent v8.5.0.

NOTE: Please merge on Thursday, December 8, 2022 (This can go out with the typical morning merge around 9:00 am pacific time)

### CODE REVIEW

**Comment by a-sassman** [comment]
@hmstepanek Can you please sign the SLA agreement? Thanks!

**Comment by a-sassman** [comment]
@bradleycamacho as the APM liason I added you as a reviewer, just FYI.

File: src/content/docs/apm/agents/python-agent/python-agent-api/recordlogevent-python-agent-api.mdx [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+title: record_log_event (Python agent API)
+type: apiDoc
+shortDescription: Records a log event for use in logging in context
+tags:
+  - Agents
+  - Python agent
+  - Python agent API
+metaDescription: 'Python API: This call records a log event for use in logging in context.'
+redirects:
+  - /docs/agents/python-agent/python-agent-api/recordlogevent-python-agent-api
```
Comment (rhetoric101) [inline src/content/docs/apm/agents/python-agent/python-agent-api/recordlogevent-python-agent-api.mdx:11]: @hmstepanek this redirect looks identical to this new file, which could cause some confusion for Gatsby. I'd like to remove it, unless you see something else going on here.

File: src/content/docs/apm/agents/python-agent/python-agent-api/recordlogevent-python-agent-api.mdx [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+  - /docs/agents/python-agent/python-agent-api/record_log_event
```
Comment (rhetoric101) [inline src/content/docs/apm/agents/python-agent/python-agent-api/recordlogevent-python-agent-api.mdx:12]: @hmstepanek usually we see redirects for deleted files, but I'm not seeing a deleted record_log_event.mdx file in this PR. Was it deleted a while back? It's not out in the wild right now.

**Review by rhetoric101 — COMMENTED** [review]
@hmstepanek I just made a few minor tweaks. Can I remove that redirect that is identical to the new page?

File: src/content/docs/apm/agents/python-agent/python-agent-api/recordlogevent-python-agent-api.mdx (hunk 1 above)
Comment (hmstepanek) [reply src/content/docs/apm/agents/python-agent/python-agent-api/recordlogevent-python-agent-api.mdx:11]: Oops. I copy/paste/modified this from another api doc and I agree I do not think these redirects need to be here. It's ok to remove them!

File: src/content/docs/apm/agents/python-agent/python-agent-api/recordlogevent-python-agent-api.mdx (hunk 2 above)
Comment (hmstepanek) [reply src/content/docs/apm/agents/python-agent/python-agent-api/recordlogevent-python-agent-api.mdx:12]: This is a new api function so there wouldn't be any pre-existing to redirect from. These can be removed.

File: src/content/docs/release-notes/agent-release-notes/python-release-notes/python-agent-80500.mdx [hunk 3]
```diff
@@ -15,7 +15,7 @@ Install the agent using `easy_install/pip/distribute` via the [Python Package In
 
 * **Add support for redis v4.3.5 & v4.4.0 methods**
```
Comment (rhetoric101) [inline src/content/docs/release-notes/agent-release-notes/python-release-notes/python-agent-80500.mdx:16]: @hmstepanek How about changing the ampersand to 'and' on line 16, too?

**Review by rhetoric101 — APPROVED** [review]
@hmstepanek thanks for making those changes! I just had one more suggestion.

File: src/content/docs/release-notes/agent-release-notes/python-release-notes/python-agent-80500.mdx (hunk 3 above)
Comment (TimPansino) [reply src/content/docs/release-notes/agent-release-notes/python-release-notes/python-agent-80500.mdx]: Fixed

**Comment by rhetoric101** [comment]
I am temporarily closing this PR so I can trigger a new build when I reopen it.

**Review by rhetoric101 — APPROVED** [review]
@hmstepanek thanks for your attention to detail on this! We can get this published on the morning of 12/8/2022.

**Pull request merged**